tasks/dependencies.rb

```diff
@@ -104,9 +104,9 @@ def berksfile_lock_task(task_name, dirs: [])
       # TODO in some edge cases, stable will actually be the latest chef because
       # promotion *moves* the package out of current into stable rather than
       # copying
-      puts "Getting latest chef 'current' version from omnitruck ..."
+      puts "Getting latest chef 'stable' version from omnitruck ..."
       options = {
-        channel: :current,
+        channel: :stable,
```

Comment:
> 1. Remove the `conservative` parameter altogether from chef and chef-dk
2. Rename update_current_chef to update_chef
3. Add a parameter to the task, `channel`, which defaults to `stable` at the moment since that’s what we want

The conservative stuff was confusing me. @jkeiser indicates we don't really need it anymore, so we should pull it (and remove the reference from the README).